src/util/colorScale.js

```diff
@@ -391,15 +391,19 @@ function booleanColorScale(val) {
   return "#CBB742";
 }
 
+/**
+ * @param {Array<Number>} legendValues
+ * @returns {Record<Number, [Number, Number]>}
```

Comment:
I don't think I've ever seen the `Record` type. Is this a custom type?

Reply:
It's [typescript syntax](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type). 

Auspice is a hodgepodge of typescript-as-comments, JSdoc, and syntax that's probably made up. I've been writing a few little personal projects with Typescript lately and I'd love for Auspice to have it, even if we migrated file-by-file over a long time. I don't really know how to combine with JSdoc -- [maybe this?](https://wojciechkrysiak.medium.com/typescript-jsdoc-better-docs-7c03b6ea04df) -- or whether we ditch JSdoc syntax (we're not auto-building docs with it or anything).